service/worker/deletenamespace/activities.go

```diff
@@ -60,21 +65,25 @@ func NewActivities(
 	}
 }
 
-func (a *activities) GetNamespaceIDActivity(ctx context.Context, nsName namespace.Name) (namespace.ID, error) {
+func (a *activities) GetNamespaceInfoActivity(ctx context.Context, nsID namespace.ID, nsName namespace.Name) (getNamespaceInfoResult, error) {
```

Comment:
why do we need to support by ID?

Reply:
But how would they know what is their namespaceID? We would have to lookup their namespace ID by their old namespace name in the form of old_name.deleted.random. We can continue to use the renamed new name instead of namespace ID. If you look at persistence, namespaceID is used to lookup for namespace name and then from namespaces table by their name.

Reply:
Let's keep it as workflow param but not expose via API. If we found it useless I will remove it from workflow params too.